test/with-typescript/src/App.tsx

```diff
@@ -1460,3 +1460,44 @@ Multitenancy.init({
         }),
     },
 });
+
+// Testing that 'null' is allowed to be returned from getRedirectionURL
+SuperTokens.init({
```

Comment:
also add here that null is allowed to be returned from session claim's on failure redirection as well.

Reply:
Something like this:

```
import { BooleanClaim } from "supertokens-auth-react/recipe/session";

export const PhoneVerifiedClaim = new BooleanClaim({
    id: "phone-verified",
    refresh: async () => {
        // This is something we have no way of refreshing, so this is a no-op
    },
    onFailureRedirection: () => null,
});
```

Reply:
done